sdk/communication/azure-communication-email/azure/communication/email/_generated/_client.py

```diff
@@ -36,40 +34,28 @@ class AzureCommunicationEmailService(object):  # pylint: disable=client-accepts-
     :paramtype api_version: str
     """
 
-    def __init__(
-        self,
-        endpoint,  # type: str
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> None
-        _endpoint = '{endpoint}'
+    def __init__(  # pylint: disable=missing-client-constructor-parameter-credential
+        self, endpoint: str, **kwargs: Any
+    ) -> None:
+        _endpoint = "{endpoint}"
         self._config = AzureCommunicationEmailServiceConfiguration(endpoint=endpoint, **kwargs)
         self._client = PipelineClient(base_url=_endpoint, config=self._config, **kwargs)
 
-        client_models = {k: v for k, v in models.__dict__.items() if isinstance(v, type)}
-        self._serialize = Serializer(client_models)
-        self._deserialize = Deserializer(client_models)
+        self._serialize = Serializer()
+        self._deserialize = Deserializer()
         self._serialize.client_side_validation = False
-        self.email = EmailOperations(
-            self._client, self._config, self._serialize, self._deserialize
-        )
-
+        self.email = EmailOperations(self._client, self._config, self._serialize, self._deserialize)
 
-    def _send_request(
-        self,
-        request,  # type: HttpRequest
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> HttpResponse
+    def send_request(self, request: HttpRequest, **kwargs: Any) -> HttpResponse:
```

Comment:
We can expose this in the EmailClient as well.

Reply:
Actually may let's not for now - I don't want it to get confused with the existing `send` method on the client.